docs/content/references/cli/validator.mdx

```diff
@@ -37,125 +35,12 @@ Options:
       --json                    Return command outputs in json format
   -y, --yes                     
   -h, --help                    Print help
-  -V, --version                 Print version
```

Comment:
Was that removed by mistake?

Reply:
No, this is not included anymore in the output.